repmgr.conf.sample

```diff
@@ -395,12 +403,12 @@ ssh_options='-q -o ConnectTimeout=10'	# Options to append to "ssh"
 #    # this is required when running sudo over ssh without -t:
 #    Defaults:postgres !requiretty
 #    postgres ALL = NOPASSWD: /usr/bin/systemctl stop postgresql-9.6, \
-#	/usr/bin/systemctl start postgresql-9.6, \
-#	/usr/bin/systemctl restart postgresql-9.6
+#        /usr/bin/systemctl start postgresql-9.6, \
+#        /usr/bin/systemctl restart postgresql-9.6
```

Comment:
As we are here, maybe change the examples to user postgresql-16

Reply:
Done thanks for advice